packages/fxa-shared/test/db/models/auth/prune-token.spec.ts

```diff
@@ -151,35 +149,6 @@ describe('PruneTokens', () => {
     ).to.be.true;
   });
 
-  it('Ignores redundant prune requests', async () => {
```

Comment:
This was removed. Now that this is out of process, we don't need to worry about de-bouncing calls.